types/external/gfycat.ts

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
the file above `components/gif_picker/utils/constants.ts` seems redundant we can probably remove it

Reply:
I see that constant from this file is being used. Do you suggest deleting `constants.ts` file and move constants to the file where they are being used? 
Because 2 (`ItemTapAction` and `appName`) out of 3 constants used in only 1 place and the remaining one (`Tab`) used in 2 different places.